packages/@okta/vuepress-site/docs/guides/add-an-external-idp/before-you-begin/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Before you begin
+---
+You want to give your users the freedom to choose which Identity Provider that they use to sign in to your application. Okta manages connections to Identity Providers for your application, sitting between your application and the Identity Provider that authenticates your users.
+
+We also support additional services such as directories and credential providers. See the [Okta Integration Network Catalog](https://www.okta.com/okta-integration-network/) to browse all integrations by use case.
+
+This guide assumes that you:
+
+* Have an Okta Developer Edition organization. (Don't have one? [Create one for free](https://developer.okta.com/signup).)
+* Have an application that you want to add authentication to.
+
+<StackSelector snippet="prereq" />
```

Comment:
I agree with moving the provider-specific pre reqs together, especially as I go through the steps  for adding a facebook IdP. In step 1, we convey "you should have an application. don't have one? create an account here." In step 2 is actually where you are told how to create the application. I think we should have all the relevant app creation instructions together

Reply:
@ZoeMonosson-okta - Are you talking about this on the Before you begin page:

* Have an Okta Developer Edition organization. (Don't have one? [Create one for free](https://developer.okta.com/signup).)

Because that one is Okta org specific. I have moved the IdP-specific info to section 2 today. But, I think that this information above should remain in the Before you begin section. Your thoughts?

Reply:
@susanharper-okta No, I was assessing this specifically for the Facebook use case. So I was looking at what was in the Facebook tab: `Go to http://developers.com and register for a developer account...` If you moved the IdP specific info to Step 2, you've addressed my comment